Removing old config split development config as it is causing issues …

soda736 opened the pull request:
I renamed this file to avoid confusion in a previous commit but it kept 2 files with the same config. So removing the one that isn't needed.

soda736 committed:
Removing old config split development config as it is causing issues with config import

johangant reviewed (commented):
This PR removes the local dev split. Is there another one being added? Your comment implied it was being renamed; if that's correct then I think the renamed or new file is missing in the PR for this to work.

soda736 commented:
I added a local_dev split with the name of development_config.yml. I wanted to rename this so it was inline with the local_dev split so renamed local_dev.config.yml but it seemed to keep both .yml files in the code. Orla got an error when pulling the latest as both had the same UUID. So I have removed the unwanted one from development.
This is the file that has been renamed nicsdru_cybersecurity/config/sync/config_split.config_split.local_dev_config.yml

soda736 committed:
Adding information about config_split in the readme.md

soda736 committed:
Merge branch 'development' into config_split_settings

soda736 committed:
Revert "Removing old config split development config as it is causing issues with config import"

This reverts commit 6c1a7b3eab34267fac56a7fb1e108fc696936a5e.

soda736 committed:
Merge branch 'development' into config_split_settings

soda736 committed:
Renaming the config split

soda736 committed:
Merge branch 'development' into config_split_settings

soda736 commented:
I have renamed the config split setting to local_dev to avoid confusion for environments. I've also updated the Readme for config split settings to be added to the settings.php.

soda736 committed:
Merge branch 'development' into config_split_settings
